### PR TITLE
Forward compatibility with Stream v0.5

### PR DESCRIPTION
**Pull request by clue**
With some minimal changes, this component is in compatible with v0.4 and the new v0.5.

### CODE REVIEW

**Comment by clue** [comment]
Updated to include documentation about STDIO streams, so this now closes #17.

File: README.md [hunk 1]
```diff
@@ -40,6 +37,36 @@ Once a process terminates, the streams will become closed but not unset.
 * `$stdout`
 * `$stderr`
 
+Each of these implement the underlying
+[`DuplexStreamInterface`](https://github.com/reactphp/stream#duplexstreaminterface)
+and you can use any of its events and methods as usual:
+
+```php
+$process->stdout->on('data', function ($chunk) {
```
Comment (jsor) [inline README.md]: `$chunk` -> `$data`

Comment (clue) [inline README.md]: I'd rather leave this like this in order to emphasize this is not necessarily a complete "data message", but could possibly be only a small "data chunk". This is also in line with all other stream documentations.

Any objections / recommendations?

Comment (jsor) [inline README.md]: I don't care, but you need to change the line below to `echo $chunk;` :)
Sorry, for not being clear enough.

Comment (clue) [inline README.md]: Duh, thanks for spotting, fixed and squashed :shipit:

**Comment by clue** [comment]
Rebased now that #27 is in :shipit:

**Pull request merged**